Dockerfile_mbtci_java11

```diff
@@ -0,0 +1,99 @@
+FROM sapmachine:11
```

Comment:
I mentioned in the previous PR that the java 11 and java17 dockerfiles
appear to have significant duplication, perhaps this is something you will want
to tackle in the future.

E.g Java 21 will be released in ~10 months and with the current approach
this would cause even more duplication.

Reply:
yes;

we will simplify the docker file asap, make java 11 and java 17 use the same docker file template